### PR TITLE
Security & Privacy: make principle imperative, for #75

### PR DESCRIPTION
**Pull request by rhiaro**
Would like to know what members of Privacy TF think of this change.

### CODE REVIEW

**Review by torgo — APPROVED** [review]
lgtm

**Comment by hober** [comment]
Works for me.

**Review by hober — APPROVED** [review]

**Review by hadleybeeman — APPROVED** [review]
This works well

**Review by torgo — APPROVED** [review]
lgtm

File: index.html [hunk 1]
```diff
@@ -207,7 +207,7 @@ <h3 id="allpeople" data-export="" data-dfn-type="dfn">
 
 <section>
   <h3 id="privacy" data-export="" data-dfn-type="dfn">
-Security and privacy are essential
+The web must be secure, and respect peoples' privacy
```
Comment (rhiaro) [inline index.html:210]:
```suggestion
The web must be secure, and privacy-respecting
```

**Pull request merged**